SPECS/envoy/envoy.spec

```
@@ -235,14 +137,19 @@ CC=gcc CXX=g++ bazel --batch build \
     --cxxopt="-Wno-implicit-fallthrough"\
     --copt="-Wno-return-type" \
     --cxxopt="-Wno-return-type" \
+    --copt="-Wno-vla-parameter" \
+    --cxxopt="-Wno-vla-parameter" \
     --curses=no \
     --host_force_python=PY3 \
     --repository_cache=BAZEL_CACHE \
     --strip=never \
-    --override_repository="boringssl=%{_prefix}/src/boringssl/" \
     --override_repository="com_github_curl=%{_datadir}/bazel-workspaces/curl" \
     --override_repository="com_github_nghttp2_nghttp2=%{_datadir}/bazel-workspaces/nghttp2" \
     --override_repository="zlib=%{_datadir}/bazel-workspaces/zlib" \
+    --override_repository="org_golang_x_text=/usr/src/mariner/BUILD/%{name}-%{version}/external/org_golang_x_text" \
+    --override_repository="com_github_spf13_afero=/usr/src/mariner/BUILD/%{name}-%{version}/external/com_github_spf13_afero" \
+    --override_repository="com_github_lyft_protoc_gen_star=/usr/src/mariner/BUILD/%{name}-%{version}/external/com_github_lyft_protoc_gen_star" \
+    --override_repository="com_github_iancoleman_strcase=/usr/src/mariner/BUILD/%{name}-%{version}/external/com_github_iancoleman_strcase" \
```

I think we can use `%{_builddir}` here:

```suggestion
    --override_repository="org_golang_x_text=%{_builddir}/%{name}-%{version}/external/org_golang_x_text" \
    --override_repository="com_github_spf13_afero=%{_builddir}/%{name}-%{version}/external/com_github_spf13_afero" \
    --override_repository="com_github_lyft_protoc_gen_star=%{_builddir}/%{name}-%{version}/external/com_github_lyft_protoc_gen_star" \
    --override_repository="com_github_iancoleman_strcase=%{_builddir}/%{name}-%{version}/external/com_github_iancoleman_strcase" \
```